cyber/tools/cyber_monitor/general_channel_message.cc

```diff
@@ -98,6 +98,13 @@ double GeneralChannelMessage::frame_ratio(void) {
   return frame_ratio_;
 }
 
+double GeneralChannelMessage::band_width(void) {
```

Comment:
Remove `void`.

Reply:
Other member functions also have void. Is it inconsistent that the bandwidth function does not have void?

Reply:
for example: double GeneralChannelMessage::frame_ratio(void)

Reply:
now band_width function is not necessary for redefinition because its implementation is the same with parent class.